src/tcp_sock.rs

```diff
@@ -1,3 +1,12 @@
+// Copyright 2018 MaidSafe.net limited.
```

Comment:
for UDP the file is udp.rs but for TCP it's tcp_sock.rs. Can you pls rename this to tcp.rs too ?